h/templates/activity/search.html.jinja2

```diff
@@ -82,6 +82,14 @@
 
   {{ panel('navbar') }}
 
+  {% for url in asset_urls("search_js") %}
+    <script src="{{ url }}" defer></script>
```

Comment:
If you use `defer` here you'll also need to add it to other scripts which depend on this one - ie. the "site_js" bundle. The browser guarantees to execute deferred scripts in the order that it encounters them, but only once parsing of all HTML (including parsing and evaluation of any "ordinary" blocking `<script>` tags) is complete.

Hence the console error message about `require('unorm')` failing in this branch.